src/python-fastui/fastui/components/__init__.py

```diff
@@ -56,7 +56,7 @@ class Text(pydantic.BaseModel, extra='forbid'):
 
 
 class Paragraph(pydantic.BaseModel, extra='forbid'):
-    text: str
+    text: '_t.List[_t.Union[str, FormattedText, Link]]'
```

Comment:
```suggestion
    text: '_t.Union[str, _t.List[_t.Union[str, FormattedText, Link]]]'
```

you need to support just `str` as well.

Then you need to update `Paragraph` and `Button` to render this.

I guess `Text` should also be changed.